pkg/pci/pci_test.go

```diff
@@ -72,37 +74,35 @@ func TestPCIReadConfigRegister(t *testing.T) {
 			pci: PCI{
 				FullPath: dir,
 			},
-			offset:   2,
-			size:     64,
-			valsWant: 0x7766554433221100,
-			errWant:  "EOF",
+			offset: 2,
+			size:   64,
+			err:    io.ErrUnexpectedEOF,
 		},
 		{
 			name: "wrong size",
 			pci: PCI{
 				FullPath: dir,
 			},
-			offset:  0,
-			size:    0,
-			errWant: "only options are 8, 16, 32, 64",
+			offset: 0,
+			size:   0,
+			err:    ErrBadWidth,
 		},
 		{
 			name: "config file does not exist",
 			pci: PCI{
 				FullPath: "d",
 			},
-			errWant: "no such file or directory",
+			err: os.ErrNotExist,
 		},
 	} {
 		t.Run(tt.name, func(t *testing.T) {
-			if vals, got := tt.pci.ReadConfigRegister(tt.offset, tt.size); got != nil {
-				if !strings.Contains(got.Error(), tt.errWant) {
-					t.Errorf("ReadConfig() = %q, want to contain: %q", got, tt.errWant)
-				}
-			} else {
-				if vals != tt.valsWant {
-					t.Errorf("ReadConfig() = '%#x', want: '%#x'", vals, tt.valsWant)
-				}
+			vals, got := tt.pci.ReadConfigRegister(tt.offset, tt.size)
+			if !errors.Is(got, tt.err) {
+				t.Errorf("ReadConfig() = got %v, want %v", got, tt.err)
```

Comment:
Should this be combined as `t.Fatalf(... same...)`

Reply:
done, thanks!